tag.js

```diff
@@ -27,18 +27,17 @@ const opts = nomnom
     })
     .parse();
 
-return index.tagTemplate({
+index.tagTemplate({
     name: opts.name,
     tag: opts.tag,
     version: opts.version
 })
     .then((result) => {
-        if (!opts.json) {
-            console.log(
-                `Template ${result.name}@${result.version} was successfully tagged as ${result.tag}`
-            );
-        } else {
+        if (opts.json) {
```

Comment:
Is there any reason for switching the logic here? Just wondering
It seemed to be consistent with the rest of the files

Reply:
No, it's just something I forgot to change when refactoring from the `--delete` flag interface to the one we've decided upon now. Nice catch 👍